.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   },
   parserOptions: {
     ecmaVersion: 6,
+    sourceType: 'module',
```

Comment:
I wonder if we're getting these eslint errors because it's applying this setting to ALL library files:

`Error:   46:25  error  Unexpected 'this'  no-invalid-this`

That seems likely. Can we set this only for the module style files in `/examples/js/`?

Reply:
Need to research https://eslint.org/docs/latest/use/configure/. 👍